src/index.ts

```diff
@@ -23,7 +30,18 @@ interface Rules {
   'type-enum': Array<string[] | number | string>;
 }
 
-const defaultConfig = { severity: 'fail' };
+const defaultConfig = {
+  severity: 'fail' as const,
+  messageReplacer: ({ ruleOutcome, commitMessage }) => {
```

Comment:
nit: Let's declare this function outside the object, and then reference it here.